geocruncher/main.py

```diff
@@ -21,7 +20,7 @@ def run_geocruncher(args):
     model = GeologicalModel(args[3])
     model.topography = txt_extract(args[4])
     box = model.getbox()
-    
+
     if args[1] == 'all':

```

Comment:
By the way: Usage of arguments should be improved. I have opened a task for this: https://trello.com/c/kQUVpxHE/702-geocruncher-better-handling-of-command-line-arguments